src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java

```diff
@@ -143,4 +150,38 @@ public void shouldThrowExceptionOnInvalidJsonWithTypeReference() {
         processor.fromJson("{asads:", new TypeReference<List<Response>>() {
         });
     }
+
+    @Test
+    public void shouldConvertMapToObject() {
```

Comment:
seems to be converting to a ccdcase not an object here

Reply:
done